app/main.py

```diff
@@ -14,4 +14,7 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    phrase_lower = phrase.lower()
+    count_letter = phrase_lower.count(letter.lower())
+    return count_letter
```

Comment:
You could implement this solution without additional variables and in one line using chaining.